src/main/java/com/vendasta/vax/VAXCredentials.java

```diff
@@ -49,7 +49,7 @@
     }
 
     @Override
-    public void applyRequestMetadata(MethodDescriptor<?, ?> methodDescriptor, Attributes attributes, Executor executor, final MetadataApplier metadataApplier) {
+    public void applyRequestMetadata(RequestInfo requestInfo, Executor executor, MetadataApplier metadataApplier) {
```

Comment:
its been so long since i've looked at this but, who calls this method? is this going to break every sdk if they use the latest version

Reply:
We provide this class to the google libs and they call it, our SDKs don't